airbyte-cdk/python/unit_tests/sources/declarative/auth/test_session_token_auth.py

```diff
@@ -129,56 +128,55 @@ def test_get_token_username_password(requests_mock):
 
     token = SessionTokenAuthenticator(
         config=config_username_password,
-        options=options,
+        parameters=parameters,
         api_url=input_instance_api_url,
         username=input_username,
         password=input_password,
         session_token=input_session_token,
         header=header,
         session_token_response_key=session_token_response_key,
         login_url=login_url,
-        validate_session_url=validate_session_url
+        validate_session_url=validate_session_url,
     ).token
     assert token == "some session id"
 
 
 def test_check_is_valid_session_token(requests_mock):
-    requests_mock.get(f"{config['instance_api_url']}user/current",
-                      json={"common_name": "common_name", "last_login": "last_login"})
+    requests_mock.get(f"{config['instance_api_url']}user/current", json={"common_name": "common_name", "last_login": "last_login"})
 
     assert SessionTokenAuthenticator(
         config=config,
-        options=options,
+        parameters=parameters,
         api_url=input_instance_api_url,
         username=input_username,
         password=input_password,
         session_token=input_session_token,
         header=header,
         session_token_response_key=session_token_response_key,
         validate_session_url=validate_session_url,
-        login_url=login_url
+        login_url=login_url,
     ).is_valid_session_token()
 
 
 def test_check_is_valid_session_token_unauthorized():
     assert not SessionTokenAuthenticator(
         config=config,
-        options=options,
+        parameters=parameters,
         api_url=input_instance_api_url,
         username=input_username,
         password=input_password,
         session_token=input_session_token,
         header=header,
         session_token_response_key=session_token_response_key,
         login_url=login_url,
-        validate_session_url=validate_session_url
+        validate_session_url=validate_session_url,
     ).is_valid_session_token()
 
 
 def test_get_new_session_token(requests_mock):
-    requests_mock.post(f"{config['instance_api_url']}session", headers={"Content-Type": "application/json"},
-                       json={"id": "some session id"})
+    requests_mock.post(f"{config['instance_api_url']}session", headers={"Content-Type": "application/json"}, json={"id": "some session id"})
```

Comment:
not 100% on this one, its right on the final max length of a line, this was automated from the format command though, i'm more surprised that it didn't get flagged in our CI because that should rely on the format command I ran